clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -2,141 +2,16 @@ import {
   ComponentType,
   CONSENT_COOKIE_NAME,
   ConsentMethod,
+  ConsentMechanism,
+  UserConsentPreference,
   FidesCookie,
-  LegacyConsentConfig,
-  PrivacyNotice,
   LastServedNoticeSchema,
   ConsentOptionCreate,
 } from "fides-js";
-import {
-  ConsentMechanism,
-  EnforcementLevel,
-  FidesOptions,
-  PrivacyExperience,
-  UserConsentPreference,
-  UserGeolocation,
-} from "fides-js/src/lib/consent-types";
-import { FidesEndpointPaths } from "fides-js/src/services/fides/api";
-
-enum OVERRIDE {
```

Comment:
moved this over to `stubs.ts` so that my new cypress test could share these handy functions